spec/integration/plugins/pg_streaming_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+RSpec.describe "Plugins / :pg_streaming", seeds: true do
+  with_adapters do
```

Comment:
I believe this could simply use `with_adapters(:postgres) do ... end` without the need for that extra `if`